product_unique_serial/tests/test_for_unicity.py

```diff
@@ -173,10 +173,7 @@ def test_1_1product_1serialnumber_2p_in(self):
         self.transfer_picking(
             picking_1,
             self.env.ref('product_unique_serial.serial_number_demo_1'))
-        with self.assertRaisesRegexp(
-                except_orm,
-                r"you will have a quantity of '2.0' "
-                r"in lot '86137801852514'"):
+        with self.assertRaises(exceptions.Warning):
```

Comment:
@suniagajose ever is better use: `assertRaisesRegexp` to get the real traceback message